CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 8.2.1

```

Comment:
let's add that to the changes of 8.2.0 since it hasn't been released yet. I will release it this weekend or start of next week.